src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Truck truck = new Truck();
```

Comment:
be more attentive, you missed the point from the checklist
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop